SmartDeviceLinkTests/DevAPISpecs/SDLLifecycleManagerSpec.m

```diff
@@ -341,6 +345,52 @@ + (void)configure:(Configuration *)configuration {
                     expect(readyHandlerError.userInfo[NSLocalizedFailureReasonErrorKey]).to(match(response.info));
                 });
             });
+            
+            context(@"when the register response is of another language", ^{
```

Comment:
Note that additional tests are necessary for the update class (see the configuration classes for examples of how to do this).

Additionally, note that these tests are not currently run. I'll have to make a PR that fixes this by re-enabling most of the tests in the `LifecycleManagerSpec`

Reply:
Actually the update class wouldn't need to be tested as there's no code to test.  `SDLLifecycleConfigurationUpdate.m` is pretty much empty. Adding an initializer would technically require test cases (trivial code should be ignored though). Question: Is it worth to go that route?

Reply:
Because I think the initializer should be added, that's all that the test class would cover. It's saved my butt before.

Reply:
Sorry.  I don't get the point. You're asking me to add an initializer that does not do anything convenient but just plain set all the parameters which then requires new unit tests?

I don't think the initializer should be added because it's just straight forward asking for all the parameters. It doesn't shorten anything.

Reply:
This is a blocker before it can be merged. Please make the requested change.